oppia/oppia-android
# Fixes part of #1460: Shift test topics to json files

## Explanation


This PR shifts the contents of `First Test Topic` and `Second Test Topic` to json files. During this transition, there have been lot of minor changes like `exploration-id` was fixed for consistency, `thumbnails` were changed to have variety of images, etc which has led to a lot of minor changes. For example, following ids have been changed as mentioned below:
1. `TEST_EXPLORATION_ID_5` -> `TEST_EXPLORATION_ID_0`
2. `TEST_EXPLORATION_ID_6` -> `TEST_EXPLORATION_ID_2`
3. `TEST_EXPLORATION_ID_30` -> `TEST_EXPLORATION_ID_3`
4. `TEST_EXPLORATION_ID_7` -> `TEST_EXPLORATION_ID_4`
5. `TEST_EXPLORATION_ID_8` -> `TEST_EXPLORATION_ID_5`
6. `First Topic` has been renamed to `First Test Topic` (This was an inconsistency which you can notice in current app, check the title of topic in home screen and then in toolbar to TopicActivity)
7. `Second Topic` has been renamed to `Second Test Topic`  (This was an inconsistency which you can notice in current app, check the title of topic in home screen and then in toolbar to TopicActivity)

Now, such changes have led to changes in multiple files across app and domain layer.

## How to test
1. First proof that this implementation is correct is the fact that all domain layer test cases are passing with a minor changes.
2. Before checking-in in this branch, check the entire app on develop, including concept-cards (described below), maybe take screenshots for better comparison later on.
3. Now run this app and check the entire app once again, like HomeScreen, Promoted Story Cards, Recently Played Stories (click on View All), 4 tabs in topic, exploration-player, question-player, revision-card in fractions (only 1st revision card will work), concept card (described below).
4. Checking concept card - for this, in manifest file, you will need to set ConceptCardFragmentTestActivity as launcher activity.

You will notice that most changes in this branch which appears in the app is change in images which is fine, along with that this PR has led to two bonus advantages:
1. Now even the test topics show `download size` in `Topic-Info`
2. In current develop, except `Prototype Exploration`, no other exploration works correctly and leads to blank screen, this has been fixed in this PR.


## Checklist
